ideascube/mediacenter/tests/test_views.py

```diff
@@ -10,7 +10,32 @@
 from .factories import DocumentFactory
 from ideascube.search.models import Search
 
-pytestmark = pytest.mark.django_db
+# Almost every call to the mediacenter view need a catalog.
```

Comment:
"almost"

Just worried it might have an impact on the execution time of the tests... but then we're doing much worse with the various storage directories so maybe I shouldn't worry about this too much.